2019/pt-br/src/0xa8-injection.md

```diff
@@ -0,0 +1,88 @@
+API8:2019 Injeção
+=================
+
+| Agentes/Vetores | Fraquezas de Segurança | Impactos |
+| - | - | - |
+| Específico da API : Explorabilidade **3** | Prevalência **2** : Detecção **3** | Técnico **3** : Específico do negócio |
+| Atacantes vão entregar à API dados maliciosos a quaisquer vetores de injeção disponíveis (ex.: dados, parâmetros, integrações e etc.), esperando que estes dados sejam entregues a um interpretador. | Falhas por injeção são muito comuns e geralmente encontrados em consultas SQL, LDAP ou noSQL, comando em SO, *parsers* de XML e ORM. São falhar simples de encontrar ao revisar código fonte, e atacantes podem utilizar *scanners* e *fuzzers*. | Injeção pode levar ao vazamento de informação e perda de dados. Também podem levar à DoS ou a perda completa de um *host*. |
```

Comment:
fix
* where it's written "_São **falhar** simples de encontrar (...)_"
* it should be "_São **falhas** simples de encontrar (...)_"